Authentication/AuthenticationBootstrapper.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-/**
+/*
```

Comment:
Why did you change this? If you remove the second *, the user of this class will not be able to see the description.